src/dialogs/image-cropper-dialog/image-cropper-dialog.ts

```diff
@@ -0,0 +1,125 @@
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+  query,
+  unsafeCSS,
+  internalProperty,
+  PropertyValues,
+} from "lit-element";
+import "../../components/ha-dialog";
+import { haStyleDialog } from "../../resources/styles";
+import type { HomeAssistant } from "../../types";
+// @ts-ignore
+import cropperCss from "cropperjs/dist/cropper.css";
+import Cropper from "cropperjs";
+import { HaImageCropperDialogParams } from "./show-image-cropper-dialog";
+import "@material/mwc-button/mwc-button";
+
+@customElement("image-cropper-dialog")
+export class HaImagecropperDialog extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @internalProperty() private _params?: HaImageCropperDialogParams;
+
+  private _cropper?: Cropper;
```

Comment:
Another thing we should do style wise in my opinion is property that use the `@property` or `@internalProperty` should be before the others. Public being before any private properties of course

Just my opinion. Happy to hear others or just that we shouldn't care /shrug

Reply:
Yeah, I usually follow that.